iree/samples/simple_embedding/CMakeLists.txt

```diff
@@ -232,6 +248,7 @@ iree_bytecode_module(
     "-iree-input-type=mhlo"
     "-iree-mlir-to-vm-bytecode-module"
     "-iree-hal-target-backends=vulkan-spirv"
```

Comment:
don't target both - create a _cuda target that is just going for cuda - that makes it easier to disable based on whether the CUDA backend is enabled or not (otherwise there's no way to have a build that is just vulkan or just cuda).